src/createServer.js

```diff
@@ -1,9 +1,41 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
```

Comment:
you need to create normalizedUrl differently because pathname is not always correct and thats why this test fails 
![image](https://github.com/user-attachments/assets/a632ff6c-f29e-40d1-9655-f6f40c7f1545)

Reply:
I don't get the difference, isn't it just about the Node version?